packages/react-components/src/components/Drawer/DrawerMenuItem.tsx

```diff
@@ -66,58 +76,103 @@ export const DrawerMenuItem = (props: _DrawerMenuItemProps): JSX.Element => {
     props.onItemClick && props.onItemClick(ev, props.itemKey);
   const onKeyPress = (ev: React.KeyboardEvent<HTMLElement>): void => onClick && submitWithKeyboard(ev, onClick);
+  /* @conditional-compile-remove(reaction) */
+  const resources = props.reactionResources;
+  /* @conditional-compile-remove(reaction) */
+  const emojiResource: Map<string, string | undefined> = new Map([
+    ['like', resources?.likeReaction?.url],
+    ['heart', resources?.heartReaction?.url],
+    ['laugh', resources?.laughReaction?.url],
+    ['applause', resources?.applauseReaction?.url],
+    ['surprised', resources?.surprisedReaction?.url]
+  ]);
+  /* @conditional-compile-remove(reaction) */
+  const emojis: string[] = ['like', 'heart', 'laugh', 'applause', 'surprised'];
 
   const secondaryIcon = props.secondaryIconProps ? (
     <MenuItemIcon {...props.secondaryIconProps} />
   ) : props.subMenuProps ? (
     <MenuItemIcon iconName="ChevronRight" />
   ) : undefined;
 
-  return (
-    <FocusZone shouldFocusOnMount={props.shouldFocusOnMount}>
+  if (props.itemKey === 'reactions') {
+    /* @conditional-compile-remove(reaction) */
+    return (
       <Stack
-        tabIndex={0}
+        id={props.id}
         role="menuitem"
         horizontal
         className={mergeStyles(
           drawerMenuItemRootStyles(theme.palette.neutralLight, theme.fonts.small),
           props.disabled ? disabledDrawerMenuItemRootStyles(theme.palette.neutralQuaternaryAlt) : undefined,
           props.styles?.root
         )}
-        onKeyPress={props.disabled ? undefined : onKeyPress}
-        onClick={props.disabled ? undefined : onClick}
-        tokens={menuItemChildrenGap}
-        id={props.id}
       >
-        {props.iconProps && (
-          <Stack.Item
-            role="presentation"
-            styles={props.disabled ? { root: { color: theme.palette.neutralTertiaryAlt } } : undefined}
-          >
-            <MenuItemIcon {...props.iconProps} />
-          </Stack.Item>
-        )}
-        <Stack.Item styles={drawerMenuItemTextStyles} grow>
-          <Text styles={props.disabled ? { root: { color: theme.palette.neutralTertiaryAlt } } : undefined}>
-            {props.text}
-          </Text>
-        </Stack.Item>
-        {props.secondaryText && (
-          <Stack.Item styles={drawerMenuItemTextStyles} className={mergeStyles(secondaryTextStyles)}>
-            <Text
-              styles={{
-                root: { color: props.disabled ? theme.palette.neutralTertiaryAlt : theme.palette.neutralSecondary }
-              }}
+        <div style={mobileViewMenuItemStyle()}>
+          {emojis.map((emoji, index) => {
+            const resourceUrl = emojiResource.get(emoji.toString());
+
+            return (
+              <IconButton
+                key={index}
+                onClick={() => {
+                  props.onReactionClick?.(emoji);
+                }}
+                style={mobileViewEmojiStyles(resourceUrl ? resourceUrl : '', 'running')}
```

Comment:
js tip: can use null coalescence operator here:
```suggestion
                style={mobileViewEmojiStyles(resourceUrl ?? '', 'running')}
```